src/OWML.ModHelper.Interaction/ModInteraction.cs

```diff
@@ -65,12 +66,41 @@ public IList<IModBehaviour> GetDependencies(string uniqueName)
 			return _dependencyDict[uniqueName];
 		}
 
+		public IModBehaviour TryGetMod(string uniqueName)
+			=> _modList.FirstOrDefault(m => m.ModHelper.Manifest.UniqueName == uniqueName);
+
+		private object TryGetApi(string uniqueName)
+		{
+			var mod = TryGetMod(uniqueName);
+			return mod == default ? default : mod.Api;
```

Comment:
i think this would work: `return mod?.Api;`
not important tho, it's fine as it is

Reply:
mod is a ModBehaviour, right? MonoBehaviours have the unity lifetime check thing, which doesn't work with optional chaining